ra_aid/tools/memory.py

```
@@ -28,14 +28,13 @@ class SnippetInfo(TypedDict):
 _global_memory: Dict[
     str,
     Union[
-        List[Any],
         Dict[int, str],
         Dict[int, SnippetInfo],
         int,
         Set[str],
         bool,
         str,
-        int,
+        List[str],
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)